.github/workflows/ci.yml

```
@@ -11,10 +11,17 @@ jobs:
       uses: actions/setup-ruby@v1
       with:
         ruby-version: 2.7.x
+    - uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-ruby2.7.x-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
     - name: Lint with Rubocop
       run: |
         gem install bundler:1.17.3
         bundle update
+        bundle config path vendor/bundle
```

Should we be running `bundle config path vendor/bundle` before `bundle update`? 
```suggestion
        bundle config path vendor/bundle
        bundle update
```
Although, I'm not clear on the background why we were calling `bundle update` and then `bundle install`...